GameDataMiner.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
This file does not contain new code. Old code was copied out of SportAlert.php to keep things organized.